src/Abc.Zebus/CommandResult.cs

```diff
@@ -1,25 +1,30 @@
-﻿using Abc.Zebus.Util.Extensions;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using Abc.Zebus.Util.Extensions;
 
 namespace Abc.Zebus
 {
     public class CommandResult
     {
+        [Obsolete("Use the constructor with the responseMessage parameter")]
```

Comment:
User code should not instantiate `CommandResult`s - and when it does it should *probably* be updated. Marking methods as obsolete is fine for a minor version, removing them requires a major version.